src/librustc_mir/borrow_check.rs

```diff
@@ -444,13 +455,14 @@ impl<'c, 'b, 'a: 'b+'c, 'gcx, 'tcx: 'a> MirBorrowckCtxt<'c, 'b, 'a, 'gcx, 'tcx>
                      context: Context,
                      lvalue_span: (&Lvalue<'tcx>, Span),
                      kind: (ShallowOrDeep, ReadOrWrite),
-                     flow_state: &InProgress<'b, 'gcx, 'tcx>) {
+                     flow_state: &InProgress<'b, 'gcx, 'tcx>) -> bool {
```

Comment:
Can you add a comment to the function indicating what the return value means?

Reply:
I see there is no existing comment, either! How about something like this:

Checks an access to the given lvalue to see if it is allowed. Examines the set of borrows that are in scope, as well as which paths have been initialized, to ensure that (a) the lvalue is initialized and (b) it is not borrowed in some way that would prevent this access.

Returns true if an error is reported, false otherwise.

Reply:
Resolved this, thanks.